course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -0,0 +1,125 @@
+describe('Unit testing PeopleController', function() {
+    var $controller, $rootScope, $routeParams, courseInstances, $compile, djangoUrl,
+        $httpBackend, $window;
+    var controller, scope;
+
+    beforeEach(function() {
+        module('CourseInfo');
+        inject(function(_$controller_, _$rootScope_, _$routeParams_, _courseInstances_,
+                        _$compile_, _djangoUrl_, _$httpBackend_, _$window_) {
+            $controller = _$controller_;
+            $rootScope = _$rootScope_;
+            $routeParams = _$routeParams_;
+            courseInstances = _courseInstances_;
+            $compile = _$compile_;
+            djangoUrl = _djangoUrl_;
+            $httpBackend = _$httpBackend_;
+            $window = _$window_;
+
+            // this comes from django_auth_lti, just stub it out so that the $httpBackend
+            // sanity checks in afterEach() don't fail
+            $window.globals = {
+                append_resource_link_id: function(url) { return url; },
+            };
+        });
+        scope = $rootScope.$new();
+        $routeParams.course_instance_id = 1234567890;
+    });
+
+    // DI sanity check
+    it('should inject the providers we requested', function() {
+        [$controller, $rootScope, $routeParams, courseInstances, $compile,
+         djangoUrl, $httpBackend, $window].forEach(function(thing) {
+            expect(thing).not.toBeUndefined();
+            expect(thing).not.toBeNull();
+        });
+    });
+
+    describe('$scope setup', function() {
+        beforeEach(function() {
+            controller = $controller('PeopleController', {$scope: scope });
+        });
+
+        it('should set the course instance id', function() {
+            expect(scope.course_instance_id).toEqual($routeParams.course_instance_id);
+        });
+
+        it('should set dtColumns up', function() {
+            expect(scope.dtColumns).not.toBeUndefined();
+            expect(scope.dtColumns).not.toBeNull();
+        });
+
+        it('should set dtOptions up', function() {
+            expect(scope.dtOptions).not.toBeUndefined();
+            expect(scope.dtOptions).not.toBeNull();
+        });
+
+        it('should have a null dtInstance', function() {
+            expect(scope.dtInstance).toBeNull();
+        });
+    });
+
+    describe('setTitle', function() {
+        var ci;
+        beforeEach(function() {
+            ci = {
+                course_instance_id: $routeParams.course_instance_id,
+                title: 'Test Title',
+            };
+            courseInstances.instances = {};
+        });
+        afterEach(function() { courseInstances.instances = {}; });
```

Comment:
We might not...I was being paranoid about resetting the injected service, but now that I think about it, I think jasmine is injecting a new copy of it for every test...I'll confirm, and hopefully yank these.